lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -161,15 +161,20 @@ def test_user_relogin(client, server_config):
             assert data["auth_token"]
             assert response.content_type == "application/json"
             assert response.status_code == HTTPStatus.OK
+            first_auth_token = data["auth_token"]
 
             # Re-login with auth header
             time.sleep(1)
             response = client.post(
                 f"{server_config.rest_uri}/login",
-                headers=dict(Authorization="Bearer " + data["auth_token"]),
+                headers=dict(Authorization=f"Bearer {first_auth_token}"),
                 json={"username": "user", "password": "12345"},
             )
```

Comment:
Why are we providing _both_ a Bearer token _and_ username/password?  Aren't those redundant credentials?  What does this test beyond what we test at line 181?

Reply:
Tracked in issue #3258.